CMakeLists.txt

```diff
@@ -579,6 +604,8 @@ add_executable(IntegrationTestsRunner
     "source/server/syscfg_library.cpp"
     "source/server/syscfg_resource_accessor.cpp"
     "source/server/syscfg_session_handler.cpp"
+    ${data_moniker_proto_srcs}
```

Comment:
Move this after `${calibrationoperations_restricted_grpc_srcs}` below as per my other comment.

Reply:
Updated